### PR TITLE
changelog: Add #11613 backport to 3.2, 3.3 and 3.4 changelogs

### PR DESCRIPTION
**Pull request by jpbetz**
For these backports:

- https://github.com/etcd-io/etcd/pull/11622
- https://github.com/etcd-io/etcd/pull/11623
- https://github.com/etcd-io/etcd/pull/11624

### CODE REVIEW

**Comment by jpbetz** [comment]
cc Wenjia Jingyi Hu Yuchen Zhou Gyuho Lee

File: CHANGELOG-3.3.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The minimum recommended etcd versions to run in **production** are 3.2.28+, 3.3.
 
 ## v3.3.18 (2019-11-26)
 
+### etcd server
+
+- [Fix corruption bug in defrag](https://github.com/etcd-io/etcd/pull/11613).
```
Comment (gyuho) [inline CHANGELOG-3.3.md]: Oh, wait, are we backporting this to 3.3.19?

Comment (jpbetz) [reply CHANGELOG-3.3.md]: oops, fixing

Comment (jpbetz) [reply CHANGELOG-3.3.md]: Done

**Comment by jpbetz** [comment]
I'll merge this once all the backports are merged.

**Pull request merged**